src/db/sysdb_ops.c

```diff
@@ -3690,8 +3690,9 @@ int sysdb_delete_custom(struct sss_domain_info *domain,
         break;
 
     default:
-        DEBUG(SSSDBG_CRIT_FAILURE, "LDB Error: %s(%d)\nError Message: [%s]\n",
-                  ldb_strerror(ret), ret, ldb_errstring(domain->sysdb->ldb));
+        DEBUG(SSSDBG_CRIT_FAILURE,
+              "ldb_delete failed: %s(%d)\nError Message: [%s]\n",
+              ldb_strerror(ret), ret, ldb_errstring(domain->sysdb->ldb));
```

Comment:
If I understand it correctly using `\n` inside the debug message would lead to a line in the log file without the usual prefix. I'm not sure how it will look when the logs are handled by journald.  I think it would be best to just avoid it.

Reply:
It was in the original code. But I fixed this here and in another place in the same file.